File: assignment2/earth_electrons.py
#Copyright 2017 Wu Sun sunw5@bu.edu

a = 1086832411937628837875003797140300000000 / 0x3fffffff
b = 8694659295501030703000030377122600000000 / 0x3fffffff
c = (a+b) / 2
c = c * 100000000
b = b * 100000000
a = a * 100000000
print(c)
print(a)
print(b)

File: assignment2/earth_electrons.cpp
//Copyright 2017 Wu Sun sunw5@bu.edu

#include<iostream>

using namespace std ;


int main()
{
	double a,b,c ;
	int i ;
	a = 4.554862024517909 ;
	b = 1.0121915610039797 ;
	c = 8.097532488031837 ;
	for (i=1;i<=38;i++)
	{
		a = a * 10 ;
		b = b * 10 ;
		c = c * 10 ;
		}

    cout << a << endl ;
    cout << b << endl ;
    cout << c << endl ;
    return(0) ;
    }
